documentation/sphinx/source/api-c.rst

```diff
@@ -295,6 +295,12 @@ See :ref:`developer-guide-programming-with-futures` for further (language-indepe
 
    |future-get-return1| |future-get-return2|.
 
+.. function:: fdb_error_t fdb_future_get_int64(FDBFuture* future, int64_t* out)
+
+   Extracts a version from an :type:`FDBFuture` into a caller-provided variable of type ``int64_t``. |future-warning|
```

Comment:
This references version, and to match the style of other functions we could call the parameter something like `out_value`.